spec/main_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+books = ['dummy'] # add finished books to this array
+
+RSpec.describe 'books' do
+  before(:all) do
```

Comment:
I think it'd be better to do the baking within the spec.  Here if one of the recipes fails, no specs will run.  I'd rather a broken recipe just break its one spec.